lib/build.js

```diff
@@ -99,7 +102,7 @@ class BuildModel extends BaseModel {
 
         delete this.user;
         const factory = UserFactory.getInstance();
-        const user = factory.get({ username: this.username });
+        const user = factory.get({ username: this.username, scmContext: this.scmContext });
```

Comment:
Thank you for your confirmation.
As you say, `user()` seems not to be used. So, I will remove this.